data/json/items/comestibles/carnivore.json

```diff
@@ -246,7 +249,7 @@
     "quench": -1,
     "fun": 5,
     "proportional": { "price": 6, "calories": 0.95 },
-    "flags": [ "EATEN_HOT" ]
+    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ]
```

Comment:
Is this SMOKABLE flag added by accident? Accepted and not accepted at the same time.

Reply:
No, not by accident. It follows this stages: meat-->smoked meat-->dehydrated meat. The middle stage is both smokable and smoked. Code itself allows it to be smoked further.